i18n/en.yml

```
@@ -98,7 +98,9 @@ warning_message: "WARNING"
 
 # Logging
 log_application_commands_synced: "Application commands synced successfully."
+log_auto_update_completed: "Auto update completed. Next update: {next_update}"
 log_auto_update_completed: "Automatic graph update completed. Next update: {next_update}"
+log_auto_update_started: "Auto update started"
 log_auto_update_started: "Automatic graph update started."
```

yamllint

[error] duplication of key "log_auto_update_started" in mapping

(key-duplicates)